spec/support/devise.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 RSpec.configure do |config|
-  config.include Devise::TestHelpers, type: :controller
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
Think this was updated in another PR?

Reply:
in `spec/rails_helper.rb` it was,
I missed this change in this file https://github.com/department-of-veterans-affairs/gibct-data-service/pull/390/files